### PR TITLE
Add BTC OSM proxy

### PR DESCRIPTION
**Pull request by banteg**
no tests for it yet, but here are some sanity checks
```bash
$ BUD=0xeaa474cbffa87ae0f1a6f68a3aba6c77c656f72c
$ BTC_OSM=0xf185d0682d50819263941e5f4EacC763CC5C6C42
$ MEDIANIZER=0x9B8Eb8b3d6e2e0Db36F41455185FEF7049a35CaE
$ seth call -F $BUD $BTC_OSM "peek()(uint256,bool)"
10775434000000000000000
true
$ seth call -F $BUD $BTC_OSM "peep()(uint256,bool)"
10840000000000000000000
true
$ seth call $MEDIANIZER "price(string)(uint)" BTC
10782515000
```

### CODE REVIEW

**Comment by banteg** [comment]
Deployed at [`0x82c93333e4E295AA17a05B15092159597e823e8a`](https://etherscan.io/address/0x82c93333e4e295aa17a05b15092159597e823e8a#code)

**Comment by banteg** [comment]
Submitted MakerDAO [proposal here](https://forum.makerdao.com/t/mip10c9-sp10-whitelist-yearn-finance-on-btcusd-oracle/4192).

**Review by orbxball — APPROVED** [review]
pytest looks so neat!
the linter for OSMedianizer on interface is weird lol.

File: tests/configuration/test_oracles.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import pytest
+from brownie import OSMedianizer
```
Comment (fubuloubu) [inline tests/configuration/test_oracles.py]: I don't think you need this, Brownie autoloads this as a fixture

**Pull request merged**